stock_barcodes_picking_batch/wizard/stock_barcodes_read_picking_batch_views.xml

```diff
@@ -0,0 +1,344 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_stock_barcodes_read_picking_batch_form" model="ir.ui.view">
+        <field name="name">stock.barcodes.read.picking.batch.form</field>
+        <field name="model">wiz.stock.barcodes.read.picking</field>
+        <field name="inherit_id" ref="stock_barcodes.view_stock_barcodes_read_form" />
+        <field name="mode">primary</field>
+        <field name="priority">60</field>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='message_type']" position="before">
+                <field
+                    name="candidate_picking_batch_ids"
+                    mode="kanban"
+                    attrs="{'invisible': [('candidate_picking_batch_ids', '=', [])]}"
+                    nolabel="1"
+                    options='{"always_reload": True}'
+                    force_save="1"
+                >
+                    <kanban>
+                        <field name="name" />
+                        <field name="date" />
+                        <field name="state" />
+                        <field name="picking_batch_id" />
+                        <field name="wiz_picking_batch_id" />
+                        <field name="scan_count" />
+                        <field name="is_pending" />
+                        <field name="product_ref_count" />
+                        <field name="product_ref_done" />
+                        <templates>
+                            <t t-name="kanban-box">
+                                <div
+                                    t-att-class="'oe_kanban_color_alert' + (record.is_pending.raw_value == false ? ' bg-success' : '')"
```

Comment:
Avoid dark green on green so the info is more readable
```suggestion
                                    t-att-class="'oe_kanban_color_alert' + (record.is_pending.raw_value == false ? ' bg-success text-white' : '')"
```

Also I suspect that `oe_kanban_color_alert` doesn't have any effect...

Reply:
Done!!